docs: release note 1.1.3 (#977)
symbol/symbol-explorer: CHANGELOG.md

The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

## [v1.1.3][v1.1.3] - 23-Nov-2021

Package  | Version  | Link
---|---|---
REST Core| v2.4.0 | [catapult-rest][catapult-rest@v2.4.0]
SDK Core| v1.0.3 | [symbol-sdk][symbol-sdk@v1.0.3]

### Added
- Support new transaction type (revokable) [#959](https://github.com/symbol/symbol-explorer/issues/959)
- Testnet theme [#971](https://github.com/symbol/symbol-explorer/pull/971)

### Fixed
- Invalid prop: type check failed for prop "transactionType" [#963](https://github.com/symbol/symbol-explorer/issues/963)
- Unknown custom element [#964](https://github.com/symbol/symbol-explorer/issues/964)

> - Partial transaction can not view in Explorer [#972](https://github.com/symbol/symbol-explorer/issues/972)

Reviewer: ```suggestion
- Partial transactions cannot be viewed in Explorer [#972](https://github.com/symbol/symbol-explorer/issues/972)
```

[catapult-server@v1.0.2.0]: https://github.com/symbol/catapult-client/releases/tag/v1.0.2.0
[catapult-server@v1.0.3.0]: https://github.com/symbol/catapult-client/releases/tag/v1.0.3.1

[symbol-sdk@v0.23.0]: https://www.npmjs.com/package/symbol-sdk/v/0.23.0
[symbol-sdk@v0.23.3-alpha]: https://www.npmjs.com/package/symbol-sdk/v/0.23.3-alpha-202102181227
[symbol-sdk@v0.23.3]: https://www.npmjs.com/package/symbol-sdk/v/0.23.3
[symbol-sdk@v1.0.0]: https://www.npmjs.com/package/symbol-sdk/v/1.0.0
[symbol-sdk@v1.0.1]: https://www.npmjs.com/package/symbol-sdk/v/1.0.1
[symbol-sdk@v1.0.2]: https://www.npmjs.com/package/symbol-sdk/v/1.0.2
[symbol-sdk@v1.0.3]: https://www.npmjs.com/package/symbol-sdk/v/1.0.3

[catapult-rest@v2.3.0]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.0
[catapult-rest@v2.3.3]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.3
[catapult-rest@v2.3.4]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.4
[catapult-rest@v2.3.6]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.6
[catapult-rest@v2.3.7]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.7

> [catapult-rest@v2.3.8]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.8

Reviewer 2: i don't think you need to show diffs for any of dependencies. just diffs for _this_ project.

Author: right, i will remove all of this 👍🏼
